tests/components/hand-controls.test.js

```diff
@@ -0,0 +1,153 @@
+/* global assert, process, setup, suite, test */
+var entityFactory = require('../helpers').entityFactory;
+var controllerComponentName = 'hand-controls';
+
+suite(controllerComponentName, function () {
+  setup(function (done) {
+    var el = this.el = entityFactory();
+    el.setAttribute(controllerComponentName, '');
+    process.nextTick(function () {
```

Comment:
`nextTick` is apparently needed for DOM updates to take effect before next test portion is run, as mentioned in A-Frame test docs (and experimentally confirmed, i.e. didn't work without)